packages/cubejs-databricks-jdbc-driver/src/helpers.ts

```diff
@@ -15,26 +15,37 @@ async function fileExistsOr(
 
 export async function resolveJDBCDriver(): Promise<string> {
   return fileExistsOr(
-    path.join(process.cwd(), 'DatabricksJDBC42.jar'),
+    path.join(process.cwd(), `databricks-jdbc-${OSS_DRIVER_VERSION}-oss.jar`),
     async () => fileExistsOr(
-      path.join(__dirname, '..', 'download', 'DatabricksJDBC42.jar'),
+      path.join(__dirname, '..', 'download', `databricks-jdbc-${OSS_DRIVER_VERSION}-oss.jar`),
       async () => {
         const pathOrNull = await downloadJDBCDriver();
         if (pathOrNull) {
           return pathOrNull;
         }
         throw new Error(
-          'Please download and place DatabricksJDBC42.jar inside your ' +
+          `Please download and place databricks-jdbc-${OSS_DRIVER_VERSION}-oss.jar inside your ` +
           'project directory'
         );
       }
     )
   );
 }
 
-export function extractUidFromJdbcUrl(jdbcUrl: string): string {
-  const { pathname } = new URL(jdbcUrl);
-  const [_, ...params] = pathname.split(';');
-  const searchParams = new URLSearchParams(params.join('&'));
-  return searchParams.get('UID') || 'token';
+/**
+ * Extract if exist UID and PWD from URL and return UID, PWD and URL without these params
+ * @param jdbcUrl
+ */
+export function extractAndRemoveUidPwdFromJdbcUrl(jdbcUrl: string): [string, string, string] {
```

Comment:
Please name positions in result tuple. Like this: `[a: string, b: number]`. This does not introduce actual names to fields, it's the same type, but at least can serve as a comment/documentation.
Or even better: turn it into struct with named fields.